src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -133,6 +133,47 @@ private static ClientInfo CreateClientInfo(InputClient ns, SourceInputModel? sou
             return new ClientInfo(ns.Name, clientName, clientNamespace, clientDescription, existingType, operations, clientParameters, resourceParameters);
         }
 
+        private static IReadOnlyList<InputOperation> UpdateOperations(IReadOnlyList<InputOperation> originalOperations)
+        {
+            var operations = new List<InputOperation>(originalOperations.Count);
+            foreach (var operation in originalOperations)
+            {
+                if (operation.Paging != null)
```

Comment:
What if we check for existing `maxCount` parameter right here? E.g.:
```cs
if (operation.Paging != null && !operation.Parameters.Any(p => p.Name.Equals("maxCount", StringComparison.OrdinalIgnoreCase)))
```

Reply:
@AlexanderSher That's correct.